packages/skeleton-svelte/src/lib/components/Rating/Rating.svelte

```diff
@@ -0,0 +1,134 @@
+<script lang="ts">
+	import type { RatingProps } from './types.js';
+
+	let {
+		value = $bindable(0),
+		max = 5,
+		interactive = false,
+		fraction = 1,
+		// Root
+		base = 'flex',
+		width = 'w-full',
+		text = 'text-token',
```

Comment:
The `-token` classes no longer exist in v3. We should also avoid hardcoding values whenever possible.

If you were going automatically set these I'd opt for something like:

> Current means inherit from the nearest parent, aka "use the default"

```ts
text = 'text-current',
fill = 'fill-current',
```

Optionally, when providing set values, always use a pairing like: `fill-surface-950-50`.

However, users are supplying their own icons, so perhaps they should be responsible for these values? If so, drop these two props. Honestly this would be my recommendation. Less is more.